development/tools/cmd/yamlmerge/README.md

```diff
@@ -0,0 +1,30 @@
+# YAML merge
+
+## Overview
+
+This command-line tool enables merging YAML files on a given path into one file. It simply creates a`starter.yaml` file from the component configurations under the `/prow/cluster/components` path, and generates a file that includes the content of `/prow/cluster/starter.yaml`.
```

Comment:
```suggestion
This command line tool enables merging yaml files into one single file. For the operation to work, the yaml files must follow the path with a given extension. For example, the tool creates a`starter.yaml` file from the component configurations residing under the `/prow/cluster/components` path, and puts the file under  `/prow/cluster/starter.yaml`. Then it generates a file out of the content of the `starter.yaml` file. 
```
Question: is it always the /prow/cluster/components path? I assume it can be a different path - just all the yaml files must have the same, so I would document this specific one as an example. Also, what kind of file is the target file? is it also a yaml file?